doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -676,6 +676,10 @@ a|The action to take if this action ever fails. Allowed values:
 * +ignore:+ Pretend the resource did not fail.
 * +block:+ Don't perform any further operations on the resource.
 * +stop:+ Stop the resource and do not start it elsewhere.
+* +demote:+ Demote the resource, without a full restart. This is valid only for
+  +promote+ actions and +monitor+ actions with a nonzero +interval+ and +role+
+  set to +Master+; for any other action, a configuration warning will be
+  logged, and the default behavior will be used.
```

Comment:
I don't know how to better word this, but it's unclear to me whether it means any promote actions, or only those with non-zero interval.  It's also unclear to me whether it means both a nonzero interval and role=Master must be set at the same time.

Reply:
I'll reword it